frontend/src/entry/overview/ExportModelCardDialog.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, DialogActions, DialogContent, Divider, Stack, Typography } from '@mui/material'
+import Dialog from '@mui/material/Dialog'
+import { useTheme } from '@mui/material/styles'
+import { Form } from '@rjsf/mui'
+import validator from '@rjsf/validator-ajv8'
+import Image from 'next/image'
+import logo from 'public/horizontal-dark.png'
+import { useMemo, useRef } from 'react'
+import { useReactToPrint } from 'react-to-print'
+import { ArrayFieldTemplate, DescriptionFieldTemplate, ObjectFieldTemplate } from 'src/Form/FormTemplates'
+import { EntryInterface, SplitSchemaNoRender } from 'types/types'
+import { widgets } from 'utils/formUtils'
+
+type EntryCardHistoryDialogProps = {
+  entry: EntryInterface
+  splitSchema: SplitSchemaNoRender
+  open: boolean
+  setOpen: (isOpen: boolean) => void
+}
+
+export default function ExportModelCardDialog({ entry, splitSchema, open, setOpen }: EntryCardHistoryDialogProps) {
```

Comment:
Consider renaming to `ExportEntryCardDialog` as it will be present on all entry overviews.